bin/ember-template-lint.js

```diff
@@ -15,23 +15,7 @@ function lintFile(linter, filePath, moduleId) {
   // TODO: swap to using get-stdin when we can leverage async/await
   let source = fs.readFileSync(toRead, { encoding: 'utf8' });
 
-  return linter.verify({ source, moduleId });
-}
-
-function expandFileGlobs(positional) {
-  let result = new Set();
-
-  positional.forEach(item => {
-    globby
-      .sync(item, {
-        ignore: ['**/dist/**', '**/tmp/**', '**/node_modules/**'],
-        gitignore: true,
-      })
-      .filter(filePath => filePath.slice(-4) === '.hbs')
-      .forEach(filePath => result.add(filePath));
-  });
-
-  return result;
+  return linter.verify({ source, moduleId, filePath });
```

Comment:
👍 - This is good. To date, we have used `moduleId` but that unfortunately strips the extension. In the future we want to use the full file path (because linting within `.js` files should be possible). This sets us up nicely for that...